libtelemetry/src/main/java/com/mapbox/android/telemetry/NavigationFasterRouteEvent.java

```diff
@@ -5,7 +5,7 @@
 
 import com.google.gson.annotations.JsonAdapter;
 
-class NavigationFasterRouteEvent extends Event implements Parcelable {
+public class NavigationFasterRouteEvent extends Event implements Parcelable {
```

Comment:
i'd suggest to tackle all `Navigation*.java` events in a separate PR

Reply:
Ok, will revert changes about Nav.